ibis/pyspark/compiler.py

```diff
@@ -8,10 +8,12 @@
 from pyspark.sql.functions import PandasUDFType, pandas_udf
 
 import ibis.common.exceptions as com
+import ibis.expr.api as ir
```

Comment:
`ir` should be `ibis.expr.types`

See:
https://github.com/ibis-project/ibis/blob/48095cc0c13ae0646bd965742f62793c63b0b23a/ibis/tests/expr/test_temporal.py#L10

Reply:
good catch